harvard-lts/CURIOSity
# Item show page: tools & related links

**Item show page: tools & related links**
* * *

**GitHub Ticket**:
* https://github.com/harvard-lts/CURIOSity/issues/77

# What does this Pull Request do?
Creates a "tools & related links" sidebar box with appropriate links, depending on the item (see ticket for a screenshot of what should look like)

# How should this be tested?

Navigate to an item detail page and check for the following links or buttons:

(NOTE: None of these fields should appear in the main sections of the item detail page, only in the sidebar area. They should also only appear if they are checked off in the Exhibit Dashboard -> Metadata -> Item details column, so it might be a good idea to toggle around with these to test.)

* Download this item button 
   * for ids: https://localhost:21407/demo/catalog/47-990013536920203941
   * for pds: https://localhost:21407/demo/catalog/47-990014631980203941
* Permalink (this should be present on all items, except for the demo exhibit; would be good to check this one across multiple exhibits): https://localhost:21407/contagion/catalog/36-990006050670203941_RADSCHL:1256323
* Harvard Art Museums: 
* HOLLIS Record: https://localhost:21407/contagion/catalog/36-990006050670203941_RADSCHL:1256323
* Finding Aid (default is set to not display, but you can check this off in the Exhibit Dashboard -> Metadata -> Item details column): 
* Finding Aid Component: 

There should also be a link centered below the viewer that says "Open full screen viewer" and opens the item in full screen.

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? NA
- integration tests? NA

# Interested parties
@phil-plencner-hl  @dl-maura @VanessaVenti
